firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -166,28 +168,26 @@ public void onFailure(@NonNull @NotNull Exception e) {
     Task<InstallationTokenResult> installationAuthTokenTask =
         firebaseInstallationsApi.getToken(true);
 
-    Tasks.whenAllComplete(installationIdTask, installationAuthTokenTask)
```

Comment:
noticed that we use `whenAllComplete`. This should actually be `whenAllSuccess` since we want it to fail if the installation task fails

Reply:
good catch!